twoliter/embedded/rpm2img

```diff
@@ -399,7 +340,7 @@ if [ "${UEFI_SECURE_BOOT}" == "yes" ] ; then
   cert_table "${SBKEYS}/vendor.cer" "${SBKEYS}/vendor.obj"
 
   # Replace the embedded vendor certificate, then sign shim with the db key.
-  objdumpcopy "${shim}" \
+  objdumpcopy "${shim}" "${ARCH}" \
```

Comment:
slightly nicer might be to provide arch-prefixed versions of this function, since objdump/objcopy are also target-prefixed:
```
"${ARCH}-objdumpcopy" "${shim}" ...

# in imghelper
aarch64-objdumpcopy() {
  objdumpcopy "aarch64" "${@}"
}

x86_64-objdumpcopy() {
  objdumpcopy "x86_64" "${@}"
}
```